rudder-core/src/main/scala/com/normation/rudder/services/policies/nodeconfig/NodeConfigurationCacheRepository.scala

```diff
@@ -65,7 +65,17 @@ case class NodeConfigurationCache(
   , nodeContextCache: Int
   , policyCache: Set[PolicyCache]
 
-)
+) {
+  // We need a method to correctly compare a NodeConfigurationCache that was serialized
+  // from a NodeconfigurationCache created from a NodeConfiguration (so without writtenDate)
+  def equalWithoutWrittenDate(other: NodeConfigurationCache) : Boolean = {
```

Comment:
I think the best solution would be to transform NodeConfigCache into Nodeconfig and compare them to actual config with equals